receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/accumulator.go

```diff
@@ -34,34 +34,60 @@ func (acc *metricDataAccumulator) getMetricsData(containerStatsMap map[string]*C
 	taskResource := taskResource(metadata)
 
 	for _, containerMetadata := range metadata.Containers {
+
+		containerResource := containerResource(containerMetadata)
+		taskResource.Attributes().ForEach(func(k string, av pdata.AttributeValue) {
+			containerResource.Attributes().Upsert(k, av)
+		})
+
 		stats, ok := containerStatsMap[containerMetadata.DockerID]
-		if !ok || stats == nil {
-			continue
+		acc.accumulate(convertContainerResource(containerResource))
```

Comment:
Does it make more sense to have this in an `else` below? Otherwise this is duped with the container metrics in the block too.

Reply:
Or I think a structure that is easier to follow is

```go
rms := convertContainerResource(containerResource)
if !isEmptyStats {
  // Convert stats and append them to `rms`
}
acc.accumulate(rms)
```

Reply:
Yes, you are right, accumulate the containerResource should be in `else`.